docs/k8s/run.sh

```diff
@@ -81,11 +82,11 @@ if [[ ! -z "${TS_EXTRA_ARGS}" ]]; then
 fi
 
 echo "Running tailscale up"
-tailscale --socket=/tmp/tailscaled.sock up ${UP_ARGS}
+tailscale --socket=${TS_SOCKET} up ${UP_ARGS}
```

Comment:
Add quotes here and below

Reply:
Oh I guess you don't need them below. Only here is fine

Reply:
resolved